src/manager.js

```diff
@@ -465,6 +465,9 @@ AdManager.prototype.setSlotTargeting = function (element, slot, adUnitConfig) {
   if (element.dataset.targeting) {
     slotTargeting = JSON.parse(element.dataset.targeting);
   }
+  if (element.dataset.adIndex) {
```

Comment:
An if/else block with above would make the precedence explicit for setting `slotTargeting`

Reply:
Whoops nevermind, I misread the code and see that it's extending the value!

Reply:
Now I'm curious, why use `utils.extend()` instead of a more-readable `slotTargeting['ad_index'] = element.dataset.adIndex` ?

Reply:
In this case, I'd agree - though, I had reached out to @Jdtanacredi separately about moving this targeting into the `data-targeting` attribute instead of introducing another requirement/change with the ad container mark-up.  So, that will likely mean the changes are isolated to Mantle as the integrating app

Reply:
Moving the value into targeting so no changes will be needed in this repo